content/applications/hr/payroll/work_entries.rst

```diff
@@ -2,173 +2,304 @@
+conflicts. Next, click the :guilabel:`Regenerate Work Entries` button at the top of the work entries
+dashboard, and a :guilabel:`Work Entry Regeneration` pop-up appears.
+
+Select the :guilabel:`Employees` to regenerate work entries for from the :guilabel:`drop-down menu`,
 and adjust the :guilabel:`From` and :guilabel:`To` fields so the correct date range is displayed.
-Click the :guilabel:`Regenerate Work Entries` button and the work entries will be recreated. Once
-finished, the pop-up will close.
+
+Click the :guilabel:`Regenerate Work Entries` button and the work entries are recreated. Once
+finished, the pop-up closes.
 
 .. image:: work_entries/regenerate-details.png
    :align: center
    :alt: Regenerate a work entry for a particular employee.
 
-Conflicts
-=========
+.. example::
+   An employee has incorrect work entries generated from the *Planning* app because they were
+   incorrectly assigned to two work stations simultaneously. This should be fixed in the *Planning*
+   app instead of the *Payroll* app.
 
-A conflict appears for any request that has not been approved, such as sick time or vacation, or if
-there are any errors on the work entry, such as required fields being left blank. Conflicts are
-required to be resolved before payslips can be generated.
+   To correct this issue, modify the employee's schedule in the *Planning* app so they are correctly
+   assigned to only one work station. Then, in the *Payroll* app, regenerate work entries for that
+   employee, for that time period.
 
-Any work entry that has a conflict to be resolved is indicated on the main :guilabel:`Work Entry`
-overview dashboard, located by going to :menuselection:`Payroll --> Work Entries --> Conflicts`,
-where only conflicts needing resolution are shown.
+   The *Payroll* app then pulls the new, corrected data form the *Planning* app, and recreates the
+   correct work entries for that employee. All conflicts for that employee are now resolved.
 
-.. image:: work_entries/conflicts.png
-   :align: center
-   :alt: Conflicts dashboard view showing all employee's conflicts in work entries.
+Generating payslips
+===================
 
-Conflicts are indicated with an orange triangle in the top left corner of each individual entry.
-Click on an individual work entry to see the conflict details in a pop-up.
+To generate payslips, :ref:`navigate to the time period <payroll/adjust-view>` the payslips should
+be generated for. Ensure the :guilabel:`Conflicting` filter is removed. When the desired pay period
+is displayed, click the :guilabel:`Generate Payslips` button.
 
-The conflict is briefly explained in an orange text box.
+.. tip::
+   If the :guilabel:`Generate Payslips` button is not active (appears pale purple instead of dark
+   purple), that indicates there are conflicts, or the date selected includes dates in the future.
+   Resolve all conflicts before generating payslips.
```

Comment:
```suggestion
   If the :guilabel:`Generate Payslips` button is not active (appears pale purple, instead of dark
   purple), that indicates there are conflicts, or the date selected includes dates in the future.
   Resolve all conflicts before generating payslips.
```